nupic/regions/PCANode.py

```diff
@@ -435,36 +435,3 @@ def getOutputElementCount(self, name):
     else:
       raise Exception('Unknown output: ' + name)
 
-
-
-if __name__=='__main__':
-
-  from nupic.engine import *
-
-  rgen = numpy.random.RandomState(37)
-
-  inputSize = 8
-
-  net = Network()
-  sensor = net.addRegion('sensor', 'py.ImageSensor' ,
-          '{ width: %d, height: %d }' % (inputSize, inputSize))
```

Comment:
@scottpurdy ok, disregard my last comment. This PR does keep `PCANode` here where it should stay. It is just removing (already broken) example from "main". Or do you want me to just comment this out to keep the code here?